src/Commands/Down.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Stancl\Tenancy\Commands;
+
+use Illuminate\Console\Command;
+use Stancl\Tenancy\Concerns\HasATenantsOption;
+
+class Down extends Command
+{
+    use HasATenantsOption;
+
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+
+    protected $signature = 'tenancy:down
+                                 {--time= : The time when the app has been set to maintenance mode}
+                                 {--message= : Message to display}
+                                 {--retry= : The number of seconds after which the request may be retried}
+                                 {--allowed=* : List of IPs allowed}';
```

Comment:
This doesn't implement the same parameters as the default `artisan` down command, right?

For instance, the `secret` — https://github.com/laravel/framework/blob/8.x/src/Illuminate/Foundation/Console/DownCommand.php#L23